### ISSUE TITLE
Unable to locate DevToolsLauncher.exe

### ISSUE THREAD

**Issue by wietze** [LOLBAS-Project/LOLBAS#155]
When trying to validate the behaviour described in [devtoolslauncher.yml ](https://github.com/LOLBAS-Project/LOLBAS/blob/master/yml/OtherMSBinaries/Devtoolslauncher.yml) (#46) I have been unable to find the executable in question. The tweet suggests it is either Visual Studio Code or Visual Studio (both very different products), but having installed both, I still can't find the `devtoolslauncher.exe` in the `system32` folder.

I have used:
- The latest version of VSCode
- The latest version of Visual Studio 2019 Community (16.11.2) using the default installation settings plus the following modules:
  - .NET dev tools (including .NET Core runtime and .NET Dev tools)
  - UWP dev tools
  - C++ dev tools
  - Latest Windows 10 SDK
- Additionally, I installed the [Remote Tools](https://visualstudio.microsoft.com/downloads#remote-tools-for-visual-studio-2019) for Visual Studio 2019. The only Google results I found on this tool point to VSRemoteTools, which is why I tried installing it. 

TL;DR I can't validate the behaviour as I can't get hold of the executable. I have no doubt it exists somewhere but if we can't work out how to get it, it is debatable whether it classifies as a Living-of-the-Land tool. 

(cc @felamos)

**Comment by bohops** [comment]
I cannot locate this lolbin either  It could very well be in an older version of VS, If it cannot be found, it may be worth removing from LOLBAS.

**Comment by nasbench** [comment]
Was looking at this and it seems that the utility was indeed available in older version. As shown by this thread https://developercommunity.visualstudio.com/t/remote-debugging-on-windows-arm64-doesnt-work/636722

But after installing a multitude of Visual Studio components, I still didn't find it. 

For ref here is a VT instance https://www.virustotal.com/gui/file/84877a507af8b70c145777a87eaf28a8327c50a1563fe650f34572bef8a42ff6/details
